### PR TITLE
Sync community templates

### PR DESCRIPTION
**Pull request by stbenjam**
Syncing the templates now so nightly keeps working after we remove all
the templates from Katello.  Katello needs the new redhat_register
snippet.

### CODE REVIEW

File: app/views/unattended/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+Gemfile
```
Comment (mmoll) [inline app/views/unattended/.gitignore]: What's the background of this?

Comment (stbenjam) [reply app/views/unattended/.gitignore]: There's a Gemfile in the community-templates repo and the sync script adds it to the repo and it's not ignored.  I could modify the sync script, instead if you'd prefer that

Comment (mmoll) [reply app/views/unattended/.gitignore]: I see, personally I would add the full path in the main .gitignore then

Comment (stbenjam) [reply app/views/unattended/.gitignore]: I'll just get rid of the gitignore entirely

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
merged, thanks @stbenjam!

**Pull request merged**